apps/projects/app/index.js

```diff
@@ -10,53 +20,6 @@ import App from './components/App/App'
 //   whyDidYouUpdate(React)
 // }
 
-// TODO: Convert to stateless functional component
-class ConnectedApp extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      app: new Aragon(new providers.WindowMessage(window.parent)),
-      network: {},
-      observable: null,
-      userAccount: '',
-    }
-  }
+// TODO: React strict
```

Comment:
Could you expand on this TODO

Reply:
I wanted to try https://reactjs.org/docs/strict-mode.html but it has lot of potential errors right now, so it should be done after we refactor to spot any missing problem there.